package.json

```diff
@@ -14,7 +14,8 @@
     "serve": "yarn build && node config/serve.js",
     "lint": "yarn eslint `find src -name '*.jsx'` `find src -name '*.js'`",
     "test": "jest",
-    "precommit": "yarn test && yarn lint"
+    "precommit": "yarn test && yarn lint",
+    "prepare": "husky install"
```

Comment:
Can we call this smth like husky-init

Reply:
The lifecycle script must be called `prepare` -- that's a special part of the Yarn lifecycle that is automatically triggered when certain things are run.

https://docs.npmjs.com/cli/v8/using-npm/scripts